department-of-veterans-affairs/caseflow
# Restructured LookupParticipantIdModal to Remove Redux Alerts

Connects #13615

### Description
This restructures the `LookupParticipantIdModal` component to remove Redux connections and re-implement success/error `Alert` within this component rather than `QueueFlowModal` (from which they were removed).

### Acceptance Criteria
- [ ] Alerts show within `LookupParticipantIdModal`
- [ ] Alerts do not show on Team Management page behind modal
- [ ] Redux-connected success Alerts do not show in other components that rely on `QueueFlowModal`

### Testing Plan
1. Go to "Caseflow Team Management"
2. Scroll down to "Private Bar" section and click "Look up Participant ID" button
3. Perform lookups and verify alert presence

### User Facing Changes

 
### Documentation Updates
- [ ] Topline File Descriptions Added or Updated as Needed
